### ISSUE TITLE
JSON Schema

### ISSUE THREAD

**Issue by nealvs** [FamilySearch/gedcomx#312]
The existing familysearch/gedcom SDKs are generic which makes them hard to use.  In order to better integrate, a JSON Schema is needed so that code can be auto-generated to match the models and create much more usable SDKs.  Parsing the github markup or html would take just as much work as manually creating code models to match the format.   http://json-schema.org/ is leading the way in this regard and is what should likely be used.  If there was a JSON Schema available, custom SDKs could be easily generated which would make integration more simple.

**Comment by stoicflame** [comment]
I agree that a JSON Schema would be helpful. If anybody would like to pick this issue up and submit a schema, I'd be happy to review it and add it to the project.

**Comment by run2000** [comment]
You might refer to https://github.com/rootsdev/gedcomx-json-schema as a starting point.

**Comment by michaelmilligan** [comment]
Even if there are no plans to create it in a relation database, has anyone yet created a logical ERD?

**Comment by MarioGravel** [comment]
I'll need this so I'm working on something from https://github.com/rootsdev/gedcomx-json-schema
